src/GafferImage/ImageReader.cpp

```diff
@@ -103,7 +104,7 @@ ImageReader::ImageReader( const std::string &name )
 	addChild( colorSpace );
 	colorSpace->inPlug()->setInput( oiioReader->outPlug() );
 	colorSpace->inputSpacePlug()->setInput( intermediateColorSpacePlug() );
-	colorSpace->outputSpacePlug()->setValue( OpenColorIO::ROLE_SCENE_LINEAR );
+	colorSpace->outputSpacePlug()->setValue( config->getColorSpace( OpenColorIO::ROLE_SCENE_LINEAR )->getName() );
```

Comment:
Could you explain this one to me Lucien? What confuses me is that we seem to be using ROLE_SCENE_LINEAR directly here, and presumably getting away with it :

https://github.com/GafferHQ/gaffer/blob/master/startup/gui/ocio.py#L72

Should we be changing that line too, or is there a reason it's not needed there? I don't doubt that you've fixed something here, but I'd like to understand the situation better...